src/style.css

```diff
@@ -1 +1,64 @@
-/* Styles go here */
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+html {
+  font-family: Roboto, Helvetica, sans-serif;
+  font-size: 12px;
+  --link-color: #00acdc;
```

Comment:
better to define variables using [root](https://developer.mozilla.org/en-US/docs/Web/CSS/:root)